packages/fast-usdc/test/cli/airdrop-commands.test.js

```diff
@@ -0,0 +1,245 @@
+
+const getBrand = brandName => walletKit =>
+  Promise.resolve(walletKit.agoricNames.brand[brandName]);
+
+const getInstanceAndBrand =
+  ({ brandName, instanceName }) =>
+  wk =>
+    Promise.all([getInstance(instanceName)(wk), getBrand(brandName)(wk)]);
+const handleFetchNetworkConfig = ({ env, fetch }) =>
+  fetchEnvNetworkConfig({ env, fetch });
+
+const handleMakeSmartWalletKit =
+  ({ delay, fetch }) =>
+  networkConfig =>
+    makeSmartWalletKit({ delay, fetch }, networkConfig);
+
+const makeClaimAirdropOffer = (
+  { feeBrand, instanceName },
+  { proof, tier, pubkey },
+) => ({
+  id: `offer-${pubkey.slice(0, 10)}`,
+  invitationSpec: {
+    source: 'agoricContract',
+    instancePath: [instanceName],
+    callPipe: [['makeClaimTokensInvitation']],
+  },
+  proposal: {
+    give: {
+      Fee: {
+        brand: feeBrand,
+        value: 5n,
+      },
+    },
+  },
+  offerArgs: {
+    proof,
+    tier,
+    pubkey,
+  },
+});
+
+const trace = label => value => {
+  console.log(label, '::::', value);
+  return value;
+};
+const traceAsync = label => value => Promise.resolve(value).then(trace(label));
+
+const composeM =
+  method =>
+  (...ms) =>
+    ms.reduce((f, g) => x => g(x)[method](f));
+
+const composePromises = composeM('then');
+
+// Successfully
+const collectPowers = config =>
+  composePromises(
+    trace('inspecting walletKit capabilities'),
+    handleMakeSmartWalletKit(config),
+    traceAsync('network config'),
+    () => handleFetchNetworkConfig({ env: config.env, fetch: config.fetch }),
+  );
+
+const addAirdropCommands = (program, { fetch, stderr, stdout }) => ({
+  operator: program.command('claimer').description('Claim airdrop commands'),
+  invoke() {
+    this.operator
+      .command('send')
+      .description(
+        'constructs an OfferSpec in accordance with makeClaimTokensInvitation from ertp-airdrop.',
+      )
+      .option('--proof <string>', 'Proof', String)
+      .option('--pubkey <string>', 'Public key', String)
+      .option('--tier <number>', 'Tier', Number)
+      .action(async opts => {
+        const { proof, tier, pubkey } = opts;
+        const powers = await collectPowers({
+          delay: 500,
+          fetch,
+          env: { AGORIC_NET: 'xnet' },
+        })();
+
+        const [instance, brand] = await getInstanceAndBrand({
+          brandName: 'IST',
+          instanceName: 'tribblesXnetDeployment',
+        })(powers);
+        console.log('------------------------');
+        console.log({ instance, brand });
+
+        /** @type {import('@agoric/smart-wallet/src/invitations.js').InvitationSpec} */
+        const offer = makeClaimAirdropOffer(
+          { feeBrand: brand, instanceName: 'tribblesXnetDeployment' },
+          { proof, tier, pubkey },
+        );
+
+        console.log({ offer });
+        outputActionAndHint(
+          { method: 'executeOffer', offer },
+          { stderr, stdout },
+          powers.marshaller,
+        );
+      });
```

Comment:
This is the sole action i'm concerned with wiring up as it facilitates claim attempts to be made programatically. 

By inspecting the logs, you will see that the code is successful in its efforts to query capabilities that exist on the chain (e.g. `instane`, `brand`) as well as their corresponding slot value. However, execution of `outputActionAndHint` appears to be where trouble arises. 

It is programmed to print a string at the end of execution but no such string appears currently thus indicating that there is an issue. Additionally, the `offer` object is receiving some complaints from VSCode intellisense. I've compared it to `OfferSpec`s that i am using to submit these very invitations to ensure that it has the correct shape but VSCode continues to say otherwise 🤷‍♂️ .